sourcegraph/about
# Workshops page

Closes #5562 creating a workshops page for workshop sign ups.

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure the form captures the data in HubSpot and you receive an email from the HubSpot workflow
